src/libcore/fmt/mod.rs

```diff
@@ -81,6 +81,21 @@ pub type Result = result::Result<(), Error>;
 /// This type does not support transmission of an error other than that an error
 /// occurred. Any extra information must be arranged to be transmitted through
 /// some other means.
+///
+/// The key thing to remember here is that the type `fmt::Error` should not be
```

Comment:
I would say "An important thing to remember is" instead, that is, this is very important, but not *the* thing, IMO